### DISCUSSION TITLE
Codeblocks in light mode

### DISCUSSION THREAD

**Discussion by beyond9thousand** [retypeapp/retype#455] [Q&A]
I noticed that the codeblocks in light mode retain the same color as they do in dark mode. I can't think of a reason for this to be intentional. If it is, shouldn't it not be this way?

I believe it should be a off-white color / light grey, as opposed to a darker shade of brown / black.
![image](https://user-images.githubusercontent.com/77363063/204080666-f305f8d1-be6b-4709-b347-e8d75e8afbe2.png)

**Comment by DannyBen** [comment]
Yes please!
